Pearson-Advance/openedx-themes
# PADV-510: Add Certiport Theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-510

## Description

This PR adds the templates and assets for Certiport theme on edx-platform.

## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-certiport-theme` on a site.
- Compile the Certiport theme.
- Verify that the Certiport theme is applied correctly on Olive.
- Enable Course Operations 
- Test course operations features

(For more information enter here: [How to enable themes)](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html#enable-themes)

This site is involved in Course Operation [(Course Operation features)](https://github.com/Pearson-Advance/course_operations/blob/master/pearson_course_operation/docs/course_operations_features_juniper.md).


## Screenshots

Log In:
![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/550f59c0-6787-49e3-876c-53f74ea9fd7f)


Dashboard:
![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/574cd5c0-247b-4bc1-a10a-f578103ea27a)

## Reviewers
